chipsec/module_common.py

```diff
@@ -36,8 +43,58 @@ class ModuleResult:
     INFORMATION = 5
     NOTAPPLICABLE = 6
     ERROR = -1
-
-
+    # -------------------------------------------------------
+    class status(Enum):
```

Comment:
Question: Is there a link to some modules that have been converted to get a better sense of how the capability will work?

Reply:
This PR includes remap.py with support